client/pages/topsecret.jsx

```diff
@@ -13,31 +63,125 @@ const HomePage = React.createClass({
+        }
+        else {
+          if(tool.name.indexOf(this.state.nameQ) > -1) {
+          return (<ReactBS.ListGroupItem> SKU: {tool.sku}, Type: {tool.type}, Name: {tool.name}, Cost: ${tool.cost}</ReactBS.ListGroupItem>);
+        }
+        }
+      }
+
+    }
+  );
+
     return (
 
       <div>
 
         <ReactBS.Panel header={<h2>Bank Account</h2>}>
-          <p style={{color: 'green'}}>Offshore Account: $4,294,967,296.69</p>
-        <p style={{color: 'red'}}>Spending Cash: $9001.75</p>
+          <p style={{color: 'green'}}>Savings Account: ${initialBankSavings.toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,')}</p>
+        <p style={{color: 'red'}}>Expenses: ${initialExpenses.toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,')}</p>
         </ReactBS.Panel>
-        <ReactBS.Panel header={<h2>Tools</h2>}>
-          <ReactBS.ListGroup>
-            <ReactBS.ListGroupItem> OZ-13MS "Dragoon" - Exoskeleton Armor</ReactBS.ListGroupItem>
-            <ReactBS.ListGroupItem> GDCB-83F "Falling Crescent" - High Frequency Glaive  </ReactBS.ListGroupItem>
-            <ReactBS.ListGroupItem> SBM-3W Mark III Pattern - Wrist Mounted Storm Bolter</ReactBS.ListGroupItem>
-            <ReactBS.ListGroupItem> VOLHUD - Exoskeleton Installed HUD and GPS </ReactBS.ListGroupItem>
-          </ReactBS.ListGroup>
+
+        <ReactBS.Panel header={<h2>Shop</h2>}>
+          <p>SKU <input type="text" value={skuin} placeholder = "Enter SKU of the tool"  onChange={this.handleChange.bind(null, 'skuin')} />
+          Type <input type="text" value={typein} placeholder = "Enter type of the tool"  onChange={this.handleChange.bind(null, 'typein')} /> </p>
+        <p>Name <input type="text" value={namein} placeholder = "Enter name of the tool"  onChange={this.handleChange.bind(null, 'namein')} />
+          Cost <input type="text" value={costin} placeholder = "Enter cost of the tool as XXXX.XX"  onChange={this.handleChange.bind(null, 'costin')} /> </p>
+        <ReactBS.Button bsStyle="primary" bsSize="medium" active onClick={this.handleAddClick}>Add Item to Armory</ReactBS.Button> {addlbl}
         </ReactBS.Panel>
 
-        <ReactBS.Button bsStyle="primary" bsSize="large" active>Primary button</ReactBS.Button>
+        <ReactBS.Panel header={<h2>Search</h2>}>
+
+        <p>Search by name <input type="text" value={nameQ} placeholder = "Enter name of the tool"  onChange={this.handleChange.bind(null, 'nameQ')} />
+        Filter by cost that is <select value={costfilter} onChange={this.handleChange.bind(null, 'costfilter')}>
+          <option value = ">"> > </option>
+          <option value = "<"> {'<'} </option>
+          <option value = "="> = </option>
+          </select>
+          <input type="text" value={costQ} placeholder = "this cost."  onChange={this.handleChange.bind(null, 'costQ')} />
+
+        </p>
+        <ReactBS.ListGroup>
+          {renderedNameSearch}
+        </ReactBS.ListGroup>
```

Comment:
Make sure your indenting is good. Anything inside ReactBS.Panel should be indented. 

If you aren't getting ESlint warnings for the indenting, your ESlint may not be configured correctly.